misc/package/Data Files/MWSE/core/mcm/components/Component.lua

```diff
@@ -87,27 +85,6 @@ end
 --- @class mwseMCMComponent.getComponent.componentData
```

Comment:
Is this type used anymore? Same about the `--- @alias mwseMCMComponentClass`? If not, both can be removed.